### ISSUE TITLE
Add descriptions to the StepByStep mode

### ISSUE THREAD

**Issue by sbozalieva** [IgniteUI/igniteui-cli#218]
### Description  
Getting to:
? Choose an action: 
  Complete & Run 
  ──────────────
❯ Add component 
  ──────────────
  Add view 
 

It is not clear enough what _Component_ and _View_ 

### Steps to reproduce  

1. Add descriptions
2. Remove lines in-between options.

### Expected result  

? Choose an action: 
    Complete & Run 
❯ Add component ........ adds customisable component
    Add view....................adds pre-build applications

**Comment by sbozalieva** [comment]
### Expected result
As discussed we need to improve the wording of the desriptions, here is another suggestion:

? Choose an action:
 Complete & Run
❯ Add components ........ basic templates
 Add views.......................scenario-based templates

**Comment by damyanpetev** [comment]
Implementation detail, but I was looking for some documentation on the inquirer.js choices and found this example for a [long list](https://github.com/SBoudrias/Inquirer.js/blob/master/examples/long-list.js). Interestingly it had a separation between the value and selection representation, so I tried this for a test:
```js
choices.push({
	name: 'Complete & Run' + chalk.gray("...........Install packages, run and open the application in the browser"),
	value: 'Complete & Run', // actual selection value
	short: 'Complete & Run' // displayed result after selection
}, {
	name: 'Add component' + chalk.gray("............Add a specific component view to the project"),
	value: 'Add component',
	short: 'Add component'
}, {
	name: 'Add view' + chalk.gray(".................Add a predefined scenario view to the project"),
	value: 'Add view',
	short: 'Add view'
});
```
And it did also pick up the color for the description part:
![image](https://user-images.githubusercontent.com/3198469/38101873-4271af54-338a-11e8-8e70-8c00bc16d72a.png) while selection only colored the normal text which is actually quite nice 💯
Also once you select something it leaves the short version:
![image](https://user-images.githubusercontent.com/3198469/38102087-e3c1fd96-338a-11e8-9720-2d50ff499ed8.png)

Basically seems we can go to town with descriptions and we can haz color (at least as much as [chalk](https://github.com/chalk/chalk) allows) 😃

**Comment by sbozalieva** [comment]
How about: :)

```
  Run ....................... install packages and run in the default browser
> Add component ............. add a specific component view (e.g a grid)
  Add scenario .............. add a predefined scenario view (e.g а dashboard)
```
